quarkus/config-api/src/main/java/org/keycloak/config/HttpOptions.java

```diff
@@ -65,7 +65,7 @@ public enum ClientAuth {
 
     public static final Option<String> HTTPS_CERTIFICATES_RELOAD_PERIOD = new OptionBuilder<>("https-certificates-reload-period", String.class)
             .category(OptionCategory.HTTP)
-            .description("Interval on which to reload key store, trust store, and certificate files referenced by https-* options. May be a java.time.Duration value, an integer number of seconds, or an integer followed by one of [ms, h, m, s, d]. Must be greater than 30 seconds. Use -1 to disable.")
+            .description("Interval on which to reload key store, trust store, and certificate files referenced by https-* options for the main server. May be a java.time.Duration value, an integer number of seconds, or an integer followed by one of [ms, h, m, s, d]. Must be greater than 30 seconds. Use -1 to disable.")
```

Comment:
This change by itself might be confusing. The other properties that serve as a default for the corresponding management property aren't similarly qualified.

Reply:
Yep - probably does not have to be there anyway.